plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergConfig.java

```diff
@@ -519,4 +520,17 @@ public IcebergConfig setMetadataCacheEnabled(boolean metadataCacheEnabled)
         this.metadataCacheEnabled = metadataCacheEnabled;
         return this;
     }
+
+    public boolean isBucketExecutionEnabled()
+    {
+        return bucketExecutionEnabled;
+    }
+
+    @Config("iceberg.bucket-execution")
```

Comment:
I'd like to see a test in this commit highlighting actually what the 🥩 of the PR is actually about.